.github/workflows/ci.yml

```diff
@@ -73,8 +75,12 @@ jobs:
       - name: Output build metadata
         id: metadata
         run: |
-          # Split the build name by '-' into module and version
-          IFS='-' read -r module version <<< "${{ matrix.build }}"
+          if [ "${{ matrix.build }}" = "wasm-wasi-component" ]; then
+            module="wasm-wasi-component"
+          else
+            # Split the build name by '-' into module and version
+            IFS='-' read -r module version <<< "${{ matrix.build }}"
+          fi
```

Comment:
I think we'll need to improve upon my hacky `language-version` idea eventually, but this works for now! 😄